# Direct slogan study: calculation check
This small companion checks saved results, not the correctness of every AI judgment. Run `analyze_light.py` first. Partial data must not be published as the completed study. Source instructions and all original model outputs are retained beside this notebook.

In [1]:
import json
from pathlib import Path
here = Path.cwd()
results = json.loads((here / 'light-results.json').read_text())
debates = json.loads((here / 'light-debates.json').read_text())
print('Status:', results['status'])
print('Reviewed:', len(debates), 'of', results['expectedDebates'])
print('Review method:', results['model'], results['reasoningEffort'], 'one reading per debate')
assert len(debates) == len({d['number'] for d in debates})


Status: complete
Reviewed: 187 of 187
Review method: gpt-6-astra low one reading per debate


In [2]:
for d in debates:
    assert d['source_words'] == sum(d[k] for k in ['theist_words', 'non_theist_words', 'unknown_words', 'excluded_words'])
    for side in ['theist', 'non_theist']:
        assert 0 <= d[side + '_protected_slogan'] <= d[side + '_unsupported']
for outcome in ['unsupported', 'protected_slogan']:
    s = results['statistics']['all'][outcome]
    assert sum(s['presence'].values()) == s['n']
    differences = [10000 * (d['theist_' + outcome] / d['theist_words'] - d['non_theist_' + outcome] / d['non_theist_words']) for d in debates if d['theist_words'] and d['non_theist_words']]
    assert abs(sum(differences) / len(differences) - s['mean_gap']) < 1e-10
    print(outcome, '| counts:', s['theist_count'], 'vs', s['non_theist_count'], '| paired average gap:', round(s['mean_gap'], 4))
print('Saved arithmetic agrees with the debate-level records.')


unsupported | counts: 128 vs 82 | paired average gap: 0.3208
protected_slogan | counts: 77 vs 19 | paired average gap: 0.4021
Saved arithmetic agrees with the debate-level records.


## Reading the checks
A verified quotation was located in the retained captions. That does not certify caption accuracy or settle its interpretation. An average rate gives each debate equal weight; pooled counts give longer contributions more weight. Resampling whole debates checks variation in the selection of recorded debates, not mistakes shared by the reader. A completed paper needs all 187 readings, resolved source conflicts, checked printed examples, and visual PDF review.